<launch>

    <arg name="working_dir" value="$(find lfd_interface)/data" />

    <node name="demonstration_storage" pkg="lfd_interface" type="demonstration_storage.py" output="screen">
        <param name="working_dir" value="$(arg working_dir)"/>
    </node>
    
</launch>
